proto/cmp/types/v1alpha/seatmap.proto

```diff
@@ -0,0 +1,171 @@
+syntax = "proto3";
+
+package cmp.types.v1alpha;
+
+import "cmp/types/v1alpha/description.proto";
+import "cmp/types/v1alpha/price.proto";
+
+// Basic representation of a seat with optional features.
+message Seat {
+  // Unique identifier, e.g., "12B", "A26"
+  string seat_id = 1;
+
+  // Seat type like STANDARD or PREMIUM etc
+  SeatType type = 2;
+
+  // Comma-separated features, e.g., "extra_leg_room,window"
+  // TODO: This field can be improved with a better enum and/or additional fields.
+  string features = 3;
```

Comment:
features should be a repeated string as a seat in a plane might have USB charger, window and be close to the toilets. A seat in a stadium can be covered or in the open air, designated to the home team supporters. a VIP seat in a concert might come with unlimited beverages, merchandise, Meet and greet with the artist
A photo with the artist
Exclusive access to a private pre-show soundcheck
Exclusive access to a backstage tour
Opportunity to shop for artist merch before doors open to the general public
A private performance or Q&A session with the artist
Invitation to a pre- or post-show party
Limited-edition and/or autographed merch
Official VIP laminated pass
On-site VIP concierge team
Free food and drinks
Dedicated VIP entrance and check-in
Of which most will be in the ActivityProductInfo as the package description

Reply:
Updated the message types and the diagram above